.github/workflows/branch_pages_deploy.yml

```diff
@@ -88,6 +87,10 @@ jobs:
               exec,
             });
 
+      - name: 'Checking out current branch'
```

Comment:
This is needed because in the previous step we checkout `gh-pages`. We want the rest of the workflow to run from the branch that triggered the workflow, not `gh-pages`.